playerstats.py

```diff
@@ -36,13 +36,15 @@ def getPlayerStats(club):
     df_topscorers = pd.DataFrame.from_dict(top_scorer)
     return df_topscorers
 
+
 def getGoalsScored(club):
     df_topscorers = getPlayerStats(club).sort_values(by=['goals'], ascending=False, ignore_index=True).head(5)
-    df_topscorers.index = df_topscorers.index+1
-    table=tabulate(df_topscorers,tablefmt='plain',colalign=['left','left'],showindex=False)
+    df_topscorers.index = df_topscorers.index + 1
```

Comment:
This can be removed if we're not showing indices

Reply:
Cool. Let it be for now. I'll remove it in the !assist commit